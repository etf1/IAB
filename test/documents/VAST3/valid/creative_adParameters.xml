<!-- <Creative>s with different <AdParameters> values -->
<VAST version="3.0">
    <Ad id="601364">
        <InLine>
            <AdSystem>Acudeo Compatible</AdSystem>
            <AdTitle>VAST 2.0 Instream Test 1</AdTitle>
            <Description>VAST 2.0 Instream Test 1</Description>
            <Error>http://myErrorURL/error</Error>

            <Impression>http://myTrackingURL/impression</Impression>
            <Creatives>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <AdParameters></AdParameters>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <AdParameters>test</AdParameters>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <AdParameters xmlEncoded="false">test</AdParameters>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <AdParameters xmlEncoded="true">test</AdParameters>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <AdParameters xmlEncoded="true"><root><sub>1</sub></root></AdParameters>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <AdParameters xmlEncoded="false"><root><sub>1</sub></root></AdParameters>
                    </Linear>
                </Creative>
            </Creatives>
        </InLine>
    </Ad>
</VAST>
