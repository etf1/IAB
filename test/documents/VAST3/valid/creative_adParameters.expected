{
  "version": "3.0",
  "ads": [
    {
      "adType": 1,
      "adTitle": "VAST 2.0 Instream Test 1",
      "description": "VAST 2.0 Instream Test 1",
      "adSystem": {
        "name": "Acudeo Compatible"
      },
      "id": "601364",
      "error": "http://myErrorURL/error",
      "impressions": [
        {
          "uri": "http://myTrackingURL/impression"
        }
      ],
      "creatives": [
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adParameters": {
            "xmlEncoded": false,
            "value": ""
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adParameters": {
            "xmlEncoded": false,
            "value": "test"
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adParameters": {
            "xmlEncoded": false,
            "value": "test"
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adParameters": {
            "xmlEncoded": true,
            "value": {
              "xmlEncoded": "true",
              "$t": "test"
            }
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adParameters": {
            "xmlEncoded": true,
            "value": {
              "xmlEncoded": "true",
              "root": [
                {
                  "sub": [
                    {
                      "$t": "1"
                    }
                  ]
                }
              ]
            }
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adParameters": {
            "xmlEncoded": false,
            "value": "<root><sub>1</sub></root>"
          }
        }
      ]
    }
  ]
}
